Forge/src/main/java/me/ferdz/placeableitems/client/model/GlassBottleFluidModel.java

```diff
@@ -0,0 +1,39 @@
+package me.ferdz.placeableitems.client.model;
+
+import me.ferdz.placeableitems.block.PlaceableItemsBlock;
+import me.ferdz.placeableitems.block.component.impl.BiPositionBlockComponent;
+import me.ferdz.placeableitems.client.AllVertexBoundingBox;
+import me.ferdz.placeableitems.init.PlaceableItemsBlockRegistry;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.client.renderer.Vector3d;
+import net.minecraft.util.Direction;
+
+/**
+ * An implementation of {@link FluidModel} for the {@link PlaceableItemsBlockRegistry#GLASS_BOTTLE}.
+ *
+ * @author Parker Hawke - Choco
+ */
+public class GlassBottleFluidModel extends FluidModel {
+
+    private static final Vector3d ROTATION_POINT = new Vector3d(8.0D, 1.0D, 8.0D);
+    private static final Vector3d UP_ROTATION_POINT = new Vector3d(8.0D, 5.0D, 8.0D);
+
+    private static final AllVertexBoundingBox BOUNDS = AllVertexBoundingBox.fromAABB(-2.5 / 16.0, 0.0D, -2.5 / 16.0, 2.5 / 16.0, 4 / 16.0, 2.5 / 16.0);
```

Comment:
What do these bounds represent? Just a square that will display the fluid's textures?

Reply:
Correct. Think of this as the model's coordinates. From (-2.5, 0, -2.5) to (2.5, 4, 2.5) relative to the point of rotation. The values are divided by 16 to be in pixel values.